packages/pointer_interceptor/pointer_interceptor_platform_interface/lib/src/pointer_interceptor_method_channel.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import 'pointer_interceptor_platform.dart';
+
+/// A default implementation of [PointerInterceptorPlatform].
+class PlaceholderPointerInterceptor extends PointerInterceptorPlatform {
```

Comment:
Since now we have some implementation for this, maybe re-naming it to "NoopPointerInterceport" or something?

Reply:
The file name also can be changed to noop_pointer_interceptor (or whatever better name you come up with)